synthtool/languages/node.py

```diff
@@ -94,10 +95,48 @@ def get_publish_token(package_name: str):
     return package_name.strip("@").replace("/", "-") + "-npm-token"
 
 
-def postprocess_gapic_library():
+def is_gapic_library():
+    # Pre-requisite #1: google-gax must be a dependency
```

Comment:
Please add a doc string.

Naive question: Can libraries that want to be built with gapic declare that same way?  This code looks like it's reverse-engineering something.

Reply:
I think we will eventually move to libraries declaring if they are GAPIC, but for now I want to simplify `synth.py` as much as possible, so preferring to make this duck test instead of asking each `synth.py` to tell if it's GAPIC or not.

This actually describes exactly what our common post-processing tool, [`compileProtos`](https://github.com/googleapis/gax-nodejs/blob/master/tools/compileProtos.ts), expects from a library to work, so it's more like a well-defined and commonly used interface rather than a reverse-engineering :)

Reply:
> This actually describes exactly what our common post-processing tool, compileProtos, expects from a library to work,

When compileProto's expectations change, this code will need to be updated, and that's a fragile maintenance burden.

I think it would help if I understood the underlying motivation and goals for these changes, so I booked a meeting this afternoon to discuss.

Reply:
Docstring added.

Reply:
Meeting accepted! Talk to you later today. Thank you!

Reply:
Changed the code as discussed on the meeting.